src/plugins/intel_cpu/src/emitters/snippets/cpu_runtime_configurator.cpp

```diff
@@ -50,5 +83,18 @@ void CPURuntimeConfigurator::update_loop_args(const std::shared_ptr<ov::snippets
     }
 }
 
+void CPURuntimeConfigurator::update_brgemms(const ov::snippets::lowered::LoopManagerPtr& loop_manager) const {
```

Comment:
Kernel executor table was specifically introduced to eliminate this kind of per-operation-type setters from the RuntimeConfigurator.
Can we move this logic to `BrgemmKernelExecutor::update_config`? 
If necessary, we can pass pointer to current LIR to `update_state()` method and store it inside the table, so all executors would have access to LIR (or loop manager only) if they need context.

We'll gonna need it anyway to update loops args from kernel exec table in the future.

Reply:
I thought about that but:
> Can we move this logic to BrgemmKernelExecutor::update_config?

No, we cannot, because we need to access to `LinearIR`

> If necessary, we can pass pointer to current LIR to update_state() method and store it inside the table, so all executors would have access to LIR (or loop manager only) if they need context.

I'm not sure about that because Kernel Executor table will again depends on LinearIR which on JIT code was generated. It means if we go to second Subgraph, we will take JIT Code and KernelExecutorTable with another LinearIR from cache. However, Shape inference was applied on another LinearIR -> KernelExecutorTable has not updated LinearIR (with outdated shapes)